### ISSUE TITLE
Klopt het

### ISSUE THREAD

**Issue by BerendWeij** [Collix/Collin-Loot-s-pirates#1]
```c

.gitignore 
```

Bestand: https://github.com/Collix/Collin-Loot-s-pirates

Klopt het dat je repos leeg is?

**Comment by Collix** [comment]
Klopt, ben vergeten dat excel bestand aan te passen. Ik heb dit project samen met Floris gemaakt dus het is dezelfde repository. 
________________________________________
